streamer/src/test/utils/postgres/postgresQueries.js

```diff
@@ -0,0 +1,147 @@
+const {environment} = require('../../../environment')
+const DB_SCHEMA = environment.DB_SCHEMA
+const totalStakeThreshold = 1000000
+const totalRewardThreshold = 1000000
+const totalRewardPointsThreshold = 20
+
+
+async function getQuery(postgres, queryString) {
+    let queryResult
+    try {
+        queryResult = await postgres.query(queryString)
+    } catch (error) {
+        console.log(error.stack)
+        throw new Error(`Error executing query`)
+    }
+    return queryResult
+}
+
+async function getMissedBlocksCount(postgres) {
+    const missedBlocksCount = await getQuery(postgres, `SELECT max(id)-count(*) as missedBlocks FROM ${DB_SCHEMA}.blocks`)
+    return missedBlocksCount.rows[0].missedblocks
```

Comment:
rows[0] possibly undefined.

Reply:
Object missedBlocksCount return -  rows: [ { missedblocks: '3923' } ] .As far as I can tell it`s not undefined

Reply:
Okay, it's correct for aggregation functions.